app/views/layouts/application.html.haml

```diff
@@ -29,6 +28,7 @@
     = javascript_include_tag 'application'
     %meta{content: asset_path("govuk-opengraph-image.png"), property: "og:image"}/
     = csrf_meta_tags
+    = render partial: 'shared/google_tag_manager_head'
```

Comment:
Was moving this to the bottom intentional?  It isn't clear from this PR _why_ this has happened.

Reply:
This was originally done when the code was moved across to the codebase. The decision to move it was based on this article http://labs.wrprojects.com/how-to-use-google-tag-manager-with-rails-and-turbolinks/ - but it's no longer necessary

Reply:
Will revert this change